styles/_base.scss

```diff
@@ -368,6 +374,43 @@ svg {
   }
 }
 
+#ov1 #doorbell #doorbell-container {
```

Comment:
Should this really be in base? I would think this should get it's own file in the module folder -or- if you think it's too small, I suppose we could start a modules/_misc file (similar to components/_misc) where module stylings that are too small for their own file could go.